packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -53,9 +72,6 @@ export default function ProfileMenu({
           href: `${webappUrl}devcard`,
         },
       },
-    ];
-
-    list.push(
```

Comment:
Not sure why this was here to being with 😅

Reply:
I think we had some items that go in-between when the user has certain feature flag. Should be good to be moved 🚀